.github/CODEOWNERS

```diff
@@ -1,11 +1,24 @@
 # See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
 
-*                                         @DataDog/apm-dotnet
-.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
-/shared/                                  @DataDog/apm-dotnet @DataDog/profiling-dotnet
+# By default every team is owner
+*                                         @DataDog/apm-dotnet @DataDog/debugger-dotnet-agent @DataDog/profiling-dotnet
+
+# Tracer
 /tracer/                                  @DataDog/apm-dotnet
 /tracer/src/Datadog.Trace/Ci/             @DataDog/apm-dotnet @DataDog/ci-app-tracers
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/ @DataDog/ci-app-tracers
 # /tracer/src/Datadog.Trace/AppSec/       @DataDog/apm-dotnet @DataDog/appsec-dotnet
+
+# Profiler
 /profiler/                                @DataDog/profiling-dotnet
+.github/workflows/profiler-pipeline.yml   @DataDog/profiling-dotnet
+
+# Debugger
+/debugger                                 @DataDog/debugger-dotnet-agent
```

Comment:
debugger-dotnet-**agent**, is that definitely the team name? 🤨 Not confusing at all...

Reply:
Yes it seems. https://github.com/orgs/DataDog/teams/debugger-dotnet-agent. I've sent the PR to Omer earlier today so I assume he'll check that part as well

Reply:
Hmm, the reason we did that is for consistency with https://github.com/orgs/DataDog/teams/debugger-java-agent. I'll discuss with fellow debugger folks if we can drop the `-agent` so that it'll be more consistent with other names.

Reply:
We agreed to change both the Java and .NET debugger team names. Fixed in 192a3b3fb8d4921d096c48f1628e49fa704498f9